docs/general/configuration_options.md

```diff
@@ -28,6 +28,31 @@ Possible values:
 
 When `false`, serializers must be explicitly specified.
 
+##### belongs_to_uses_id_on_self
+
+Enable automatic serializer lookup when using a `belongs_to` without `has_many`.
```

Comment:
> Tells the JSON:API adapter to build `belongs_to` relationship id and type using the foreign key on self, rather than loading the association.
> For example, a Comment that belongs to a user will have a `user_id` attribute. 
> This means the relationship will the call `object.user_id` instead of `object.user.id`, which can prevent unneeded queries.
> IMPORTANT: Because the association is no longer be loaded,  any `type` must come from the `belongs_to` options as `type: :comment_users` or `serializer: CommentUserSerializer` (which would call CommentUserSerializer._type, i.e. no instance methods)

or something to that effect. Examples below are good.